### PR TITLE
review

### PR DESCRIPTION
**Pull request by volkov-a-dev**

### CODE REVIEW

File: block/styles/blocks/footer/index.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    max-width: 1170px;
+    margin: 0 auto;
+    background-color: #597275;
+}
+
+.designed-by {
+    padding: 0 50px;
+    padding-bottom: 36px;
+    padding-top: 37px;
+    text-align: center;
```
Comment (volkov-a-dev) [inline block/styles/blocks/footer/index.css]: **This is not good:**
 padding: 0 50px;
 padding-bottom: 36px;
 padding-top: 37px;

**Rewrite one indent**
padding: ** ** **

File: block/styles/blocks/footer/subscription.css [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+.email-form {
+    display: block;
+}
+
+.email-form__input {
+    display: block;
+    float: left;
+    width: 80%;
+    min-width: 50px;
+    height: 40px;
```
Comment (volkov-a-dev) [inline block/styles/blocks/footer/subscription.css]: Bad:
`    height: 40px;`
Good;
`    min-height: 40px;`

File: block/styles/blocks/header/banner.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    background-position: center 3%;
+    background-size: cover;
+    background-repeat: no-repeat;
+    color: #fff;
+}
+
+.banner-wrap {
+    padding: 97px 0;
+    margin: 0 auto;
+    margin-top: -9px;
```
Comment (volkov-a-dev) [inline block/styles/blocks/header/banner.css]: Bad:
```
   margin: 0 auto;
   margin-top: -9px;
```

File: block/styles/blocks/header/banner.css [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    max-width: 500px;
+    text-align: center;
+}
+
+.banner__header {
+    margin-top: 4px;
+    display: block;
+    font-family: 'PT Sans', sans-serif;
+    text-align: center;
+    line-height: 52px;
```
Comment (volkov-a-dev) [inline block/styles/blocks/header/banner.css]: Best used dynamic sizes 
`    line-height: 52px;`

File: block/styles/blocks/header/banner.css [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    width: 100%;
+    text-decoration: underline;
+}
+
+.banner__text {
+    padding: 0 60px;
+    margin-top: 23px;
+    text-align: center;
+    font-family: 'Lato', sans-serif;
+    line-height: 26px;
```
Comment (volkov-a-dev) [inline block/styles/blocks/header/banner.css]: Best used dynamic sizes

File: block/styles/blocks/header/banner.css [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+.go-down {
+    display: inline-block;
+    margin-top: 35px;
+    width: 69px;
+    height: 69px;
+    border: none;
+    border-radius: 50%;
+    background-color: #e4551d;
+    text-align: center;
+    line-height: 69px;
```
Comment (volkov-a-dev) [inline block/styles/blocks/header/banner.css]: Best used dynamic sizes

File: block/styles/blocks/header/navigation.css [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+        padding: 0;
+        margin-top: 20px;
+        text-align: end;
+        font-size: 20px;
+        color: #e4551d;
+    }
+
+    #menu-toggler:checked ~ .navigation {
+        transform: translateX(0);
+        opacity: 1;
```
Comment (volkov-a-dev) [inline block/styles/blocks/header/navigation.css]: Remove opacity: 1

File: block/styles/blocks/header/navigation.css [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+        transform: translateY(-50%);
+    }
+
+    .navigation {
+        transform: translateX(150%);
+        transition: transform 0.5s cubic-bezier(0.77,0.2,0.05,1.0);
+        position: fixed;
+        top: 0;
+        right: 0;
+        opacity: 0;
```
Comment (volkov-a-dev) [inline block/styles/blocks/header/navigation.css]: Need to change opacity : 1

File: flexbox/styles/blocks/main/features.css [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+    background-color: #f6f6f6;
+}
+
+.features-wrap {
+    max-width: 1170px;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    overflow: hidden;
```
Comment (volkov-a-dev) [inline flexbox/styles/blocks/main/features.css]: Why is it here? ` overflow: hidden;`

File: flexbox/styles/blocks/main/staff.css [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+    width: 50%;
+    padding-left: 31px;
+    margin-bottom: 22px;
+}
+
+.member__photo {
+    flex-shrink: 0;
+    max-width: 100px;
+    max-height: 100px;
+    overflow: hidden;
```
Comment (volkov-a-dev) [inline flexbox/styles/blocks/main/staff.css]: Why is it here? ` overflow: hidden;`

File: flexbox/styles/blocks/main/staff.css [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+
+.member__info {
+    padding-left: 30px;
+    padding-right: 10px;
+}
+
+.member__info_header {
+    padding-bottom: 11px;
+    line-height: 17px;
+    font-size: 22px;
```
Comment (volkov-a-dev) [inline flexbox/styles/blocks/main/staff.css]: Well, I do not know for me is not fine. line-height should be font-size

File: block/index.html [hunk 12]
```diff
@@ -0,0 +1,388 @@
...
+            </label>
+
+            <nav class="navigation">
+                <a href="#" class="navigation__link">HOME</a>
+                <a href="#" class="navigation__link">ABOUT</a>
+                <a href="#" class="navigation__link">SERVICES</a>
+                <a href="#" class="navigation__link">PROJECT</a>
+                <a href="#" class="navigation__link">BLOG</a>
+                <a href="#" class="navigation__link">CONTACT</a>
+            </nav>
```
Comment (volkov-a-dev) [inline block/index.html]: I would do to improve accessibility so:
```
<nav class="navigation">
    <ul class="navigation__lists">
        <li class="navigation__element">
           <a href="#" class="navigation__link">HOME</a>
        </li>
         ........
    </ul>
</nav>
```

File: block/index.html [hunk 13]
```diff
@@ -0,0 +1,388 @@
...
+                Lorem ipsum dolor sit amet
+            </h2>
+            <h3 class="main-title__description">
+                Donec odio uisque volutpat mattis eros ullam malesuada erat ut turpis.
+            </h3>
+        </header>
+        <section class="features">
+            <div class="features-wrap">
+                <section class="feature">
+                    <header class="feature__header">
```
Comment (volkov-a-dev) [inline block/index.html]: I think the header is not appropriate here  - `<header > `

File: block/index.html [hunk 14]
```diff
@@ -0,0 +1,388 @@
...
+                    </header>
+                    <p class="feature__description">
+                        Morbi est est, blandit sit amet, sagittis vel, euismod vel, velit. Pellentesque egestas sem.
+                    </p>
+                    <button class="text-button">More</button>
+                </section>
+            </div>
+        </section>
+        <section class="portfolio">
+            <header class="main-title">
```
Comment (volkov-a-dev) [inline block/index.html]: I think the header is not appropriate here

File: flexbox/styles/blocks/main/features.css (hunk 9 above)
Comment (pavelLyolya) [reply flexbox/styles/blocks/main/features.css]: this hides negative margins and feature items borders

File: flexbox/styles/blocks/main/staff.css (hunk 10 above)
Comment (pavelLyolya) [reply flexbox/styles/blocks/main/staff.css]: its because of not the same photo sizes

**Pull request merged**